<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pantalla</class>
 <widget class="QWidget" name="pantalla">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pantalla</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>891</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="5">
     <widget class="QLabel" name="min1">
      <property name="text">
       <string>min 1</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QCustomPlot" name="G4" native="true"/>
    </item>
    <item row="0" column="2">
     <widget class="QCustomPlot" name="G1" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="prom2">
      <property name="text">
       <string>PS2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="L2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>S2</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="uni5">
      <property name="text">
       <string>uS5</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QCustomPlot" name="G5" native="true"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="L3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>S3</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="prom1">
      <property name="text">
       <string>PS1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QCustomPlot" name="G3" native="true"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="L4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>S4</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="L1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>S1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="prom3">
      <property name="text">
       <string>PS3</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="L5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>S5</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QCustomPlot" name="G2" native="true"/>
    </item>
    <item row="6" column="3">
     <widget class="QLabel" name="prom5">
      <property name="text">
       <string>PS5</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QLabel" name="min5">
      <property name="text">
       <string>min 5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QLabel" name="min4">
      <property name="text">
       <string>min 4</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLabel" name="mx4">
      <property name="text">
       <string>MAX 4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QLabel" name="min3">
      <property name="text">
       <string>min 3</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="uni2">
      <property name="text">
       <string>uS2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="uni3">
      <property name="text">
       <string>uS3</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QLabel" name="mx5">
      <property name="text">
       <string>MAX 5</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="uni1">
      <property name="text">
       <string>uS1</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="prom4">
      <property name="text">
       <string>PS4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLabel" name="mx3">
      <property name="text">
       <string>MAX 3</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="uni4">
      <property name="text">
       <string>uS4</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="mx1">
      <property name="text">
       <string>MAX 1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLabel" name="min2">
      <property name="text">
       <string>min 2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="mx2">
      <property name="text">
       <string>MAX 2 </string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>5 min</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>30 seg</string>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
   <slots>
    <slot>change()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
